FitTracker/app/src/main/java/com/pps/globant/fittracker/mvp/model/DataBase/UserRoomDataBase.java

```diff
@@ -0,0 +1,28 @@
+package com.pps.globant.fittracker.mvp.model.DataBase;
+
+import android.arch.persistence.room.Database;
+import android.arch.persistence.room.Room;
+import android.arch.persistence.room.RoomDatabase;
+import android.content.Context;
+
+
+@Database(entities = {User.class}, version = 1,exportSchema = false)
+public abstract class UserRoomDataBase extends RoomDatabase {
+    private static final String DB_NAME="base de datos room";
+
+    public abstract UserDao userDao();
+    private static UserRoomDataBase INSTANCE;
+
+    public static UserRoomDataBase getDatabase(final Context context) {
+        if (INSTANCE == null) {
+            synchronized (UserRoomDataBase.class) {
```

Comment:
y wrote it just as a good practice i read from https://codelabs.developers.google.com/codelabs/android-room-with-a-view/#6 
item 4

in case some other threads try to get an instance of the database, thinking in the other social networks based logins, which usually do the sign in in background into another thread. but no, its not necessary here, at least not for the moment. I will delete it